ci_framework/roles/artifacts/tasks/main.yml

```diff
@@ -44,8 +49,15 @@
     - always
   ansible.builtin.import_tasks: cleanup.yml
 
-- name: Gather CRC logs
-  ansible.builtin.import_tasks: crc.yml
-
-- name: Gather EDPM logs
-  ansible.builtin.import_tasks: edpm.yml
+- name: Gather logs when forced or tests failed
+  when: cifmw_artifacts_gather_logs | default('false') | bool or not cifmw_success_flag.stat.exists
```

Comment:
please add this one to the defaults/main.yml in the artifacts role. Doing so, we ensure people not reading the README but checking the defaults/main.yml know about that parameter, and you can also remove the `| default('false')` part.

Reply:
Done